build.go

```diff
@@ -0,0 +1,82 @@
+package phpredishandler
+
+import (
+	"os"
+
+	"github.com/paketo-buildpacks/packit/v2"
+	"github.com/paketo-buildpacks/packit/v2/scribe"
+	"github.com/paketo-buildpacks/packit/v2/servicebindings"
+)
+
+//go:generate faux --interface BuildBindingResolver --output fakes/build_binding_resolver.go
+//go:generate faux --interface ConfigParser --output fakes/config_parser.go
+//go:generate faux --interface ConfigWriter --output fakes/config_writer.go
+
+type BuildBindingResolver interface {
+	ResolveOne(typ, provider, platformDir string) (servicebindings.Binding, error)
```

Comment:
```suggestion
	ResolveOne(type, provider, platformDir string) (servicebindings.Binding, error)
```

Reply:
@sophiewigmore and I talked about this `typ` vs `type`, and how it is [defined as `typ` in `packit](https://github.com/paketo-buildpacks/packit/blob/2247967a3f873b178f6fb16c5e6411646ca0882a/servicebindings/resolver.go#L83)

I suspect it is because in some situations `type` is a reserved keyword and hence not a valid name. In any case, I'd suggest we keep the same naming as the upstream library, and move the conversation there.

Reply:
Fair enough